OneWelcomeExampleApp.xcodeproj/project.pbxproj

```diff
@@ -1610,7 +1610,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.3.0;
+				MARKETING_VERSION = 3.0.0;
```

Comment:
because 6.0.0 is version of config on TS, and the app is the next release after release 2.0.0:
https://github.com/onewelcome/example-app-ios/releases

Previously I thought that we can start over from 1.0.0, but it'd require another repo on GitHub, which does not make sense.